OpenNefia.Content/CharaMake/CharaMakeBackgroundLayer.cs

```diff
@@ -0,0 +1,147 @@
+﻿using OpenNefia.Content.UI;
```

Comment:
Maybe remove this if it's unused right now (Elona+ stuff is way farther down the line).

Reply:
removed.